types/data.d.ts

```diff
@@ -57,9 +64,9 @@ export interface IDecodedPayload {
     timestamp: number;
     ua: string;
     envelope: IEnvelope;
-    metrics: IDecodedMetric;
-    stream: IDecodedEvent[];
-    backup: IDecodedEvent[];
+    metrics: IMetric;
```

Comment:
>IMetric [](start = 13, length = 7)

Should this be IMetric[]?

Reply:
It is currently an object instead of an array. We can track it either way but since we are sending only updated metrics back, we need to send back the index too. If you look at the interface - it's basically a dictionary:
interface IMetric {
    [key: number]: number;
}